### DISCUSSION TITLE
Run the same rebalance again when a rebalance was successful.

### DISCUSSION THREAD

**Discussion by 2Fast2BCn** [cryptosharks131/lndg#44] [Ideas]
Would it be possible run the same rebalance again when a rebalance was successful? Right now you have to wait for the queue to finish before a rebalance attempt is done on a certain channel again. Usually I have between 15 and 22 channels that are being attempted by the scheduler. So it takes more then 30 minutes for a certain channel to get rebalanced a second time.

Or maybe have to option to run rebalances concurrently?

My problem is: even if I have a channel that rebalances easily it can't because of the very large and time consuming backlog of other channels that don't rebalance very well. I disabled a bunch of those difficult to rebalance channels on the scheduler and I'm considering bringing back my cronjobs but I would prefer not to ;-)

**Comment by cryptosharks131** [comment]
Done! #47
